akka-typed/src/test/scala/akka/typed/ActorContextSpec.scala

```diff
@@ -141,36 +140,33 @@ object ActorContextSpec {
           }
         case BecomeCareless(replyTo) ⇒
           replyTo ! BecameCareless
-          Actor.Stateful(
-            (ctx, message) ⇒ Actor.Unhandled,
-            (ctx, signal) ⇒ signal match {
-              case Terminated(_) ⇒ Actor.Unhandled
-              case sig ⇒
-                monitor ! GotSignal(sig)
-                Actor.Same
-            })
+          Actor.Stateful[Command]({
+            case (_, _) ⇒ Actor.Unhandled
```

Comment:
I think it would make sense to provide a prealocated function for these. Like `unhandledSignal` and `unhandledMessage`. Similar to the current `Actor.emptyBehavior`.